src/redux/reducers/transfer.js

```diff
@@ -121,6 +122,27 @@ export const transferReducers = {
         transferTargetResources: updateTargetResources(state.transferTargetResources)
       };
     },
+    [actionCreators.transfer.deselectTransferResource]: (state) => {
+      const deselectTargetResource = transferTargetResources => {
+        let sourceResources = transferTargetResources;
+        sourceResources.map(resource => {
+          resource.active = false;
```

Comment:
Is it possible to check if the resource is active, make it false, and then stop the loop?